docs/content/guides/dagster/code-references.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Code References in Dagster | Dagster Docs
+---
+
+# Linking to code in Dagster with Code References
```

Comment:
Should we mention assets in the title, since this seems to be specific to assets? Are we planning to expand this to other types of definitions?

Reply:
Good call, updated!